crates/sui-core/src/authority/execution_time_estimator.rs

```diff
@@ -51,6 +51,9 @@ pub struct ExecutionTimeObserver {
     // via consensus.
     object_utilization_tracker: LruCache<ObjectID, ObjectUtilization>,
 
+    // Sorted list of recently indebted objects, updated by consensus handler.
+    indebted_objects: Vec<ObjectID>,
```

Comment:
repeating my comment from the stacked version of this commit: any reason you didn't use a HashSet?

Reply:
took an educated guess that this might be faster due to cache locality, since it should be small?

happy to change it if you think that's dumb and premature optimization - it wasn't a very thoroughly considered decision - wdyt?

Reply:
yeah if its small it might be faster. no need to change it, its just we don't do this anywhere else as far as i know.